src/tests.rs

```diff
@@ -87,13 +98,18 @@ async fn test_disputes_offchain_disabling() -> Result<(), Error> {
     let total_disputes = honest.reports(DISPUTES_CONCLUDED_VALID).await? as u64;
 
     // wait a bit
-    sleep(Duration::from_secs(120)).await;
+    sleep(Duration::from_secs(60)).await;
```

Comment:
Hi @ordian did you think we can replace this `sleep`s with something like `wait_n_blocks(n)` and internally do something similar than `wait_for_block` but with `current + n` as goal.

Reply:
In general yes, it's a good idea to not rely on sleep. In this case, the test logic itself requires a sleep: we want to test that finality is not lagging "after some time" or "for some time". We could replace "time" with "blocks", but it wouldn't make much difference in this particular case.

Reply:
Sounds good, thanks for the feedback. I'm exploring some helpers to add to the new `sdk`.
Thx!!